pkgs/applications/networking/instant-messengers/rambox/default.nix

```diff
@@ -42,19 +42,19 @@ in stdenv.mkDerivation rec {
   ];
 
   installPhase = ''
-    patchelf --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" Rambox
-    patchelf --set-rpath "$out/share/rambox:${stdenv.lib.makeLibraryPath deps}" Rambox
+    patchelf --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" rambox
```

Comment:
@phryneas Thanks for being faster than me! :) You would also like to change ``exec = name;`` to ``exec = "rambox";`` at https://github.com/phryneas/nixpkgs/blob/947d862f92e1b5b1ce0be111968ef63d507d51cf/pkgs/applications/networking/instant-messengers/rambox/default.nix#L18 so menu item will work